### DISCUSSION TITLE
how to change HDR10 metadata that is read by tone-mapping?

### DISCUSSION THREAD

**Discussion by geextahslex** [mpv-player/mpv#14432] [Q&A]
Hi, I have a HDR file that has a HDR10 Mastering Display value of 10000 that is used for the tone-mapping. This breaks the `tone-mapping=st2094-10`
How can I change this value? So he reads 1k instead of 10k?

Thank you :)

![metadata1](https://github.com/mpv-player/mpv/assets/139760266/9056d76d-a609-4ca0-8a00-f28d294b4635)
![metadata2](https://github.com/mpv-player/mpv/assets/139760266/d4558272-de9c-464e-85e8-f8bc95b145c9)

**Comment by kasper93** [comment]
sample?

**Reply by geextahslex** [reply]
Here is the sample
[Ahsoka sample.zip](https://github.com/user-attachments/files/15960730/Ahsoka.sample.zip)

**Reply by kasper93** [reply]
You should have `--hdr-compute-peak` enabled anyway to compute the real peak. 

But indeed the metadata is off, I will look at it.

**Reply by geextahslex** [reply]
Is it possible to set this value to a fix value, so it doesn't change dynamically? I like the way it looks at 1000-->150 but this bounces around depending on the scene

**Reply by kasper93** [reply]
`--vf=format=sig-peak=4.926` will give you constant 1000 nits.

sig-peak is nits / 203.

**Reply by geextahslex** [reply]
It worked, awesome :D
This also fixed Ahsoka sample for me, so the 10k metadata must be faulty

Finally perfect tone mapping ^^
![perfect tone mapping](https://github.com/mpv-player/mpv/assets/139760266/d9fdfb91-e3ad-4e49-a650-62f051f39bdf)

**Reply by geextahslex** [reply]
Was this `vf=format=sig-peak=4.926` function added recently? Because it seems not to work on mpv v37, but works on v38
